x-pack/plugins/ml/public/application/management/breadcrumbs.ts

```diff
@@ -12,7 +12,7 @@ export function getJobsListBreadcrumbs() {
   return [
     {
       text: i18n.translate('xpack.ml.jobsList.breadcrumb', {
```

Comment:
Is it worth changing this i18n key to something like `xpack.ml.management.breadcrumb`?

Reply:
https://github.com/elastic/kibana/pull/130776/commits/ae744f23904966997a66cf0d3760f02a074e0a16